src/AppRouter.js

```diff
@@ -330,23 +330,22 @@ class AppRouter extends Component {
     }
 
     const { content } =
-        menus.find((menu) => {
-          return menu.name === "PortalMainNavLinks";
-        }) || {};
+      menus.find((menu) => {
+        return menu.name === "PortalMainNavLinks";
+      }) || {};
     const initialMenu = content;
-    
+
     const finalMenu = this.modifiedMenu(initialMenu);
-    this.setState({ navBarMenu: finalMenu})
+    console.log("I am the final menu bro", finalMenu);
```

Comment:
Can you drop the console logs once ready?

Reply:
Ooo yes, sorry, my bad 😅